global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "6.0.419",
+    "version": "8.0.204",
```

Comment:
In dev we are still on `8.0.101` - should we ensure these are aligned?

Reply:
I was thinking about that as I was making the change. We can go back now, or keep this in the feature branch as we should have this deployed by the time we need to merge.

Reply:
If 204 will be deployed soon, let's just keep it